src/main/java/org/sbml/spatial/segmentation/Inference.java

```diff
@@ -31,7 +29,9 @@ public Inference(ComputationGraph model, String path, String directory) {
 	 */
 
 	// For inferring the model
-	public void imgOut() {
+	public BufferedImage imgOut() {
```

Comment:
Would it make sense to rename this method `infer` instead of `imgOut`?

Reply:
Sure. Will make the change soon.